packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -7,6 +7,17 @@
       class="dropdown-title"
       type="button"
       :aria-label="dropdownAriaLabel"
+      @click="handleDropdown"
```

Comment:
Maybe you should add some comments here, its for issue that `tab + click` cannot open the `navlinks`.